src/main/kotlin/com/openlattice/linking/BackgroundLinkingService.kt

```diff
@@ -126,21 +127,25 @@ class BackgroundLinkingService
             try {
                 // only linking id of entity should remain, since we cleared neighborhood, except the ones
                 // with positive feedback
-                val cluster = getClusters(setOf(candidate)).entries.first()
-
-                val scoredCluster = cluster(candidate, cluster, ::completeLinkCluster)
-                if (scoredCluster.score <= MINIMUM_SCORE) {
-                    logger.error(
-                            "Recalculated score {} of linking id {} with positives feedbacks did not pass minimum score {}",
-                            scoredCluster.score,
-                            cluster.key,
-                            MINIMUM_SCORE
-                    )
-                }
+                val clusters = getClusters(setOf(candidate))
+                val clusterId = clusters.keys.first()
```

Comment:
would prefer to derive `clusterId` from `cluster` to prevent these from potentially getting out of sync in the future but otherwise lgtm